Rocket.Chat/Managers/AppManager.swift

```diff
@@ -110,6 +110,11 @@ struct AppManager {
 extension AppManager {
 
     static func changeSelectedServer(index: Int) {
+        guard index != DatabaseManager.selectedIndex else {
```

Comment:
Do we really have had all these things done while changing to the same server/index? Seems like `reloadApp` is just enough.